crf/src/main/java/org/sagebase/crf/result/CrfTaskResult.java

```diff
@@ -0,0 +1,21 @@
+/*
+ *    Copyright 2019 Sage Bionetworks
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package org.sagebase.crf.result;
+
+public class CrfTaskResult {

```

Comment:
So... Not sure I understand, sorry. This doesn't look like it subclasses anything. Is this placeholder code?